gradio/components/chatbot.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         show_share_button: bool | None = None,
         show_copy_button: bool = False,
         avatar_images: tuple[str | Path | None, str | Path | None] | None = None,
+        chat_bubble_fit_content: bool = False,
```

Comment:
I'm wondering if we can shorten the name of the parameter? Maybe `bubble_full_width` (which is default `True`)